examples/faucet/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+
+	addresscodec "github.com/Peersyst/xrpl-go/address-codec"
+	"github.com/Peersyst/xrpl-go/xrpl"
+	"github.com/Peersyst/xrpl-go/xrpl/client/websocket"
+	"github.com/Peersyst/xrpl-go/xrpl/faucet"
+)
+
+func main() {
+
+	fmt.Println("Funding wallet on testnet:")
+
+	testnetFaucet := faucet.NewTestnetFaucetProvider()
+	testnetClientCfg := websocket.NewWebsocketClientConfig().
+		WithHost("wss://s.altnet.rippletest.net:51233").
+		WithFaucetProvider(testnetFaucet)
+	testnetClient := websocket.NewWebsocketClient(testnetClientCfg)
+
+	wallet, err := xrpl.NewWallet(addresscodec.ED25519)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	balance, err := testnetClient.GetXrpBalance(wallet.ClassicAddress)
+	if err != nil {
+		balance = "0"
+	}
+
+	fmt.Println("Balance", wallet.ClassicAddress, balance)
+
+	fmt.Println("Funding wallet", wallet.ClassicAddress)
+	err = testnetClient.FundWallet(&wallet)
```

Comment:
Can you also write any string address for the FundWallet function? Or do you need to provide  `wallet` pointer?